modepy/shapes.py

```diff
@@ -199,76 +227,51 @@
 @dataclass(frozen=True)
 class Shape:
     """
-    .. attribute :: dims
+    .. attribute :: dim
     .. attribute :: nfaces
     .. attribute :: nvertices
     """
-    dims: int
+    dim: int
 
 
 @singledispatch
-def get_unit_vertices(shape: Shape):
+def biunit_vertices_for_shape(shape: Shape):
     """
-    :returns: an :class:`~numpy.ndarray` of shape `(nvertices, dims)`.
+    :returns: a :class:`~numpy.ndarray` of shape `(dim, nvertices)`.
     """
     raise NotImplementedError(type(shape).__name__)
 
 
-@singledispatch
-def get_face_vertex_indices(shape: Shape):
-    """
-    :results: a tuple of the length :attr:`Shape.nfaces`, where each entry
-        is a tuple of indices into the vertices returned by
-        :func:`get_unit_vertices`.
-    """
-    raise NotImplementedError(type(shape).__name__)
+@dataclass(frozen=True)
+class Face:
+    """Inherits from :class:`Shape`.
```

Comment:
Doesn't seem to inherit from `Shape`?

Reply:
c872de9a19cb34319be701d657060076e059d0c4